src/ringct/multiexp.cc

```diff
@@ -700,8 +696,14 @@ rct::key pippenger(const std::vector<MultiexpData> &data, const std::shared_ptr<
     }
   }
 
+  return result;
+}
+
+rct::key pippenger(const std::vector<MultiexpData> &data, const std::shared_ptr<pippenger_cached_data> &cache, const size_t cache_size, const size_t c)
+{
   rct::key res;
-  ge_p3_tobytes(res.bytes, &result);
+  const ge_p3 result_p3 = pippenger_p3(std::move(data), cache, cache_size, c);
```

Comment:
Even though this compiles, the `std::vector<MultiexpData>` is copied here, not moved. I don't know if this was the intended consequence. This happens because `data` is a `const&`.

Reply:
Since `pippenger_p3` takes ` const&`, you can just pass it as is